tactic/fin_cases.lean

```diff
@@ -0,0 +1,42 @@
+/-
+Copyright (c) 2018 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Scott Morrison
+
+Case bashing on `fin n`, for explicit numerals `n`.
+-/
+import tactic.linarith
+
+namespace tactic
+
+meta def is_numeral : expr → bool
```

Comment:
Are you sure this is what you intended? This will accept `bit0 (1+1)` as a numeral and reject `one`. I think it would be better to use `expr.is_numeral` from core.